ykrt/pass_finder/run_tests.sh

```diff
@@ -1,7 +1,24 @@
 #!/bin/sh
+# This script runs tests cases for YK and YKLUA. The aim is
```

Comment:
We generally prefer formatting comments to 80 or 100 chars rather than 60.

Reply:
addressed here: 147f01131d3ccab617b903709b6fdf561a9928f0

Reply:
Did you link the right commit there?

Reply:
sorry, updated!